src/h5web/toolbar/controls/DomainSlider/ScaledSlider.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ReactElement, Ref } from 'react';
+import { FiSkipBack, FiSkipForward } from 'react-icons/fi';
+import ReactSlider from 'react-slider';
+import { Domain, extendDomain, ScaleType } from '../../../../packages/lib';
+import { createAxisScale } from '../../../vis-packs/core/utils';
+import styles from './DomainSlider.module.css';
+import Thumb from './Thumb';
+
+const SLIDER_RANGE: Domain = [1, 100];
+
+interface Props {
+  rawDomain: Domain;
+  dataDomain: Domain;
+  disabled?: boolean;
+  isAutoMin: boolean;
+  isAutoMax: boolean;
+  onChange: (domain: Domain) => void;
+  onAfterChange: (domain: Domain) => void;
+}
+
+const EXTEND_FACTOR = 0.2;
+
+function ScaledSlider(props: Props): ReactElement {
+  const { rawDomain, dataDomain, disabled, isAutoMin, isAutoMax } = props;
+  const { onChange, onAfterChange } = props;
+
+  const sliderExtent = extendDomain(dataDomain, EXTEND_FACTOR);
+
+  const scale = createAxisScale({
+    type: ScaleType.Linear,
+    domain: sliderExtent,
+    range: SLIDER_RANGE,
+    round: true,
+  });
+
+  return (
+    <ReactSlider
+      className={styles.slider}
+      trackClassName={styles.track}
+      thumbClassName={styles.thumb}
+      thumbActiveClassName={styles.thumbActive}
+      pearling
+      min={SLIDER_RANGE[0]}
+      max={SLIDER_RANGE[1]}
```

Comment:
Does TS complain if you use `scale.range[0]` and `scale.range[1]` here ?

Reply:
Now, I should be able to (I couldn't before, because I was returning a domain scale)

Reply:
![image](https://user-images.githubusercontent.com/2936402/109618630-11166f00-7b38-11eb-8137-5b2103a271ee.png)

I find it more readable with the constant, anyway.

Reply:
I have no issue keeping the constant as well. It was out of curiosity :monocle_face: